crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -795,6 +796,19 @@ export namespace parse_output {
     createTreeCursor(): cursor.Cursor;
   }
 }
+export namespace query {
+  export interface QueryResult {
+    queryNumber: number;
+    bindings: { [key: string]: cursor.Cursor[] };
```

Comment:
I suggest using `ReadonlyMap<string, ReadonlyArray<cursor.Cursor>>` for the `bindings` type.

Reply:
Done

Reply:
But it isn't that type - it's a dumb JSObject. I'd rather not build a more complicated object to return.